test/built-ins/Promise/withResolvers/result.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2023 Peter Klecha. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Promise.withResolvers result is an object with keys "promise", "reject", and "resolve"
+esid: sec-promise.withresolvers
+features: [promise-with-resolvers]
```

Comment:
```suggestion
features: [promise-with-resolvers, Object.hasOwn]
```
unless you want to use `.hasOwnProperty` below

Reply:
thanks for this!